src/force-fullscreen.cpp

```diff
@@ -575,7 +576,7 @@ class wayfire_force_fullscreen : public wf::plugin_interface_t
 
     wf::signal_connection_t view_output_changed{[this] (wf::signal_data_t *data)
         {
-            auto signal = static_cast<wf::view_move_to_output_signal*>(data);
+            auto signal = static_cast<wf::view_moved_to_output_signal*>(data);
```

Comment:
Technically, this should be `wf::view_pre_moved_to_output_signal`, but maybe I'm just being pedantic.

Reply:
We want it to be correct, can you submit a PR?

Reply:
Will do.